src/object/conveyor_belt.cpp

```diff
@@ -128,6 +128,8 @@ ConveyorBelt::after_editor_set()
 
   if (m_length <= 0)
     m_length = 1;
+  m_sprite->set_action(dir_to_string(m_dir));
+  update_hitbox();
```

Comment:
The `update_hitbox()` line here wouldn't be needed, if `set_action` is not called on the sprite, but rather on the `ConveyorBelt` itself (`MovingSprite::set_action` will call `update_hitbox()`).

Didn't notice earlier, sorry.